docs/adr/0003-End-to-end-tests.md

```diff
@@ -0,0 +1,57 @@
+# 3. End to end tests
+
+Date: 2023-04-18
+
+## Status
+
+Accepted
+
+## Context
+
+End to end (e2e) tests only run tests against exporters `/metrics` endpoint in OpenShift today (using Prow CI). We want to run them with more cases, like against Grafana, making them more reliable.
+
+## Decision
+
+e2e tests will be more user friendly.
+
+Prow CI will run e2e tests that cover more scenarios and are more reliable.
+
+### How
+
+- We will rewrite the e2e tests script in Python.
+    - easier to write and read it
```

Comment:
That is statement which I disagree. In some cases it is easier to write and read in other it's not. We need to be careful adding such statement in ADR. I really don't like locking ourselves in one particular implementation language. Parts of the e2e may be considered better in python, parts in shell and parts in a testing framework which is available as community driven e2e test framework for dashboard that is currently used by Pelorus.

Reply:
changing it to 

- We will improve the development (writing) and understanding (readability) of e2e tests script
  - use frameworks to help us rewrite it (like Python's argparse)
  
would be right for you?

Reply:
`use frameworks where applicable to improve code (...)`